omnisafe/common/logger.py

```diff
@@ -87,8 +96,9 @@ def store(self, **kwargs):
         Provide an arbitrary number of keyword arguments with numerical
         values.
         """
+        # pylint: disable-next=invalid-name
         for k, v in kwargs.items():
-            if not (k in self.epoch_dict.keys()):
+            if k not in self.epoch_dict:
```

Comment:
Consider using `key, value` and remove the pylint comment.

Reply:
Other similar places have been changed.